qmk_firmware/keyboards/keyball/lib/keyball/keyball.c

```diff
@@ -127,6 +145,10 @@ void pointing_device_driver_init(void) {
     if (keyball.this_have_ball) {
         pmw3360_cpi_set(CPI_DEFAULT - 1);
         pmw3360_reg_write(pmw3360_Motion_Burst, 0);
+
+#ifdef POINTING_DEVICE_AUTO_MOUSE_ENABLE
+        set_auto_mouse_layer(1);
```

Comment:
Auto mouse layer should be changed for each keymaps.

Right now, all the keymaps happen to use layer 1 for the mouse, but essentially it should be able to be changed for each keymap.

I'll fix this.